memory-viz/src/cli.ts

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env node
 
-const fs = require("fs");
-const path = require("path");
-const { draw } = require("memory-viz");
-const { program } = require("commander");
-const { json } = require("node:stream/consumers");
+import * as fs from "fs";
+import * as path from "path";
+import { program } from "commander";
+import { json } from "node:stream/consumers";
+import { DrawnEntity } from "./types";
```

Comment:
This is only being imported as a type, so there's [better syntax](https://www.typescriptlang.org/docs/handbook/2/modules.html#typescript-specific-es-module-syntax) that will allow the compiler to erase the import after types are checked.